src/Symfony/Component/Security/Http/Firewall/SwitchUserListener.php

```diff
@@ -165,11 +163,9 @@ private function attemptSwitchUser(Request $request, string $username)
     /**
      * Attempts to exit from an already switched user.
      *
-     * @return TokenInterface The original TokenInterface instance
```

Comment:
the description seems value to be kept. maybe as a normal description with `@return`